tools/clang/test/CodeGenSPIRV/compute.byte-address-buffer.load.hlsl

```diff
@@ -0,0 +1,33 @@
+// Run: %dxc -T cs_6_0 -E main
+
+// CHECK: %_ptr_Uniform_uint = OpTypePointer Uniform %uint
```

Comment:
No need to duplicate the check here since we already have https://github.com/google/DirectXShaderCompiler/commit/16efe2418811da1ce55ac987c13ee5a9f5a6e844#diff-15573ab924a53cb0fd4e85ea58f9ea4d